mobile/Components/ArtworkCard.js

```diff
@@ -0,0 +1,46 @@
+// component that renders artwork cards on gallery screen
+import * as React from 'react';
+import {
+  View, Image, TouchableOpacity, ScrollView,
+} from 'react-native';
+import { Card, Title, Text, Button } from 'react-native-paper';
```

Comment:
Remove unnecessary imports

Reply:
removed title text button imports from react native paper
removed view and scrollview imports from react native